mayday/mayday.go

```diff
@@ -1,20 +1,42 @@
 package mayday
 
 import (
-	"fmt"
 	"log"
-	"os"
-	"path"
 )
 
-func maybeCreateLink(src, dst, workspace string) error {
-	if src == "" {
-		return nil
+func Run(t Tar, files []File, commands []Command, journals []Journal) error {
+	defer t.Close()
```

Comment:
I think either the tar should be created here, or this shouldn't close the tar file.  Whichever function creates the tar should be responsible for closing it.